### PR TITLE
ECO-654: General css style adjustment

### PR DESCRIPTION
**Pull request by AbnerZheng**
### Overview
This PR update the general css style basing on https://xd.adobe.com/view/ed1c8e8b-00bf-44c2-b08c-4fc2009636c9-b84f/. 

### Which JIRA ticket does this PR relate to?
https://casperlabs.atlassian.net/browse/ECO-654

### Complete this checklist before you submit this PR

- [ ] This PR contains no more than 200 lines of code, excluding test code.
- [ ] This PR meets [CasperLabs coding standards](https://casperlabs.atlassian.net/wiki/spaces/EN/pages/16842753/Coding+Standards).
- [ ] If this PR adds a new feature, it includes tests related to this feature.
- [ ] You assigned one person to review this PR.
- [ ] Your GitHub account is linked with our [Drone CI](https://drone-auto.casperlabs.io/) system. This is necessary to run tests on this PR.
- [ ] Do not forget to run `bors r+` if GitHub policy is not enforced, e.g. when merging into another feature branch. It may be omitted under some circumstances if this PR intentionally assumes that integration tests will fail but will be fixed with the future PRs.

### Notes

_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._

### CODE REVIEW

File: packages/ui/src/components/App.tsx [hunk 1]
```diff
@@ -74,7 +84,7 @@ class GroupedMenuItem {
           data-target={`#${this.id}`}
           aria-controls={this.id}
         >
-          <i className={`nav-link-icon fa fa-fw fa-${this.icon}`} />
+          <div style={{ margin: '0 10px 0 20px' }}>{this.icon}</div>
```
Comment (PfanP) [inline packages/ui/src/components/App.tsx]: Do we have any class defined already? like class m-3 in bootstrap

File: packages/ui/src/components/App.tsx [hunk 2]
```diff
@@ -220,9 +272,7 @@ const NavLink = (props: { item: MenuItem }) => {
             data-placement="right"
           >
             <Link to={item.path} className="nav-link">
-              {item.icon && (
-                <i className={'nav-link-icon fa fa-fw fa-' + item.icon} />
-              )}
+              <div style={{ margin: '0 10px 0 20px' }}>{item.icon}</div>
```
Comment (PfanP) [inline packages/ui/src/components/App.tsx]: Same as above. Maybe we can define the class for this styling if nothing similar exist

File: packages/ui/src/components/App.tsx (hunk 1 above)
Comment (AbnerZheng) [reply packages/ui/src/components/App.tsx]: Done.

File: packages/ui/src/components/App.tsx (hunk 2 above)
Comment (AbnerZheng) [reply packages/ui/src/components/App.tsx]: Good suggestion, done.

**Comment by zie1ony** [comment]
#28  already contains that code.

**Pull request closed without merging**